src/john.c

```diff
@@ -858,30 +858,30 @@ static void john_load_conf(void)
 	}
 
 	if (options.activewordlistrules == NULL)
-		if (!(options.activewordlistrules =
+	  if (!(options.activewordlistrules = (char*)
```

Comment:
Minor whitespace problem here. I used some default emacs settings in live stick Ubuntu during merge. I'll defer a fix for now.